modules/setup/includes/functions.inc.php

```diff
@@ -90,4 +90,54 @@ function write_bashScript($sys){
                     @touch('/var/www/html/LibreEHR/test.sh');
             }
 
+
+        /**
+         * Get all of the directories within a given directory.
+         *
+         * @param  string  $directory
+         * @return array
+         */
+        function getSitesDirectories($directory)
+        {
+            $glob = glob($directory . '/*');
+            if($glob === false)
+            {
+                return array();
+            }
+            return array_filter($glob, function($dir) {
+                return is_dir($dir);
+            });
+        }
+
+        function getConfigFiles($directory){
+            foreach( glob($directory."/*.php") as $filename ) {
+                $basename = basename($filename);
+                if ($basename == "sqlconf.php") {
+                    return $filename;
+                }
+            }
+        }
+
+        function findString($filename) {
+            // What to look for
+            $search1 = '$config = 0';
+            $search2 = '$config = 1';
+            // Read from file
+            $lines = file($filename);
+            foreach($lines as $line)
+            {
+                // Check if the line contains the string we're looking for, and print if it does
+                if(strpos($line, $search1) !== false){
+                    $val = substr($line,9,2);
+                    echo $val;
+                }else{
+                    if(strpos($line, $search2) !== false){
+                        $val = substr($line,9,2);
+                        echo $val;
+                    }
+                }
```

Comment:
Suceeded in getting the stuff and values now i will place the select dropdown just above this screen below.
![screen shot 2018-10-21 at 11 40 20 pm](https://user-images.githubusercontent.com/16350814/47273336-b72a3580-d58a-11e8-8f1a-bed26fae024a.png)

Also art the force installation button should ask which site you would love to delete or re-install cause you might have default, clinic that are ok but default4 which maybe you messed up with the db. And you want a re-install there. If this is ok please tell me. Will push more commits tomorrow

Reply:
The 110 are the config variables found in the default site, three and two if i am not mistaken. I placed them there to see the recursiveness will remove them once done. So here is my idea where it detects a one in the sqlconfig, it denotes it as an installed site and vice versa for the other. The 110 denotes there is an active installation in default/ and in three/ so the dropdown will comprise of these. and not of two. But with the force install you can override any directory , or wipe all something of the sort waiting for our ideas on that 😃